build.sbt

```diff
@@ -1,6 +1,10 @@
 import scala.sys.process._
+import scala.util.Try
+import BindingHelpers._
 
-addCommandAlias("verify", "; ^test:compile ; ^test ; ^scripted ; docs/makeSite")
+addCommandAlias(
+  "verify",
+  "; tests/test ; samples/test ; bindings/test ; ^scripted ; docs/makeSite")
```

Comment:
I had to break it up to run the tests in separate commands. It might make sense to reorganize the `tests` directory into sub-projects so it will be `; tests/test ; bindings/test ; ...`